spec.md

```diff
@@ -42,7 +41,7 @@ Typically, these manifests may provide different implementations of the image, p
 
 ![](img/build-diagram.png)
 
-Once built the OCI Image can then be discovered by name, downloaded, verified by hash, trusted through a signature, and unpacked into an [OCI Runtime Bundle](https://github.com/opencontainers/runtime-spec/blob/master/bundle.md).
+Once built the OCI image can then be discovered by name, downloaded, verified by hash, trusted through a signature, and unpacked into an [OCI Runtime Bundle](https://github.com/opencontainers/runtime-spec/blob/master/bundle.md).
```

Comment:
I see `OCI Image` used throughout the spec, so I don't know if I agree with this line.

Reply:
There are also a lot of lowercase - I guess I don't see it as a proper noun so I made it lowercase. But if you are requesting to make it capital again, happy to do that.

Reply:
All set.